README.md

```diff
@@ -32,9 +32,11 @@ The module can be configured with several optional parameters passed to it's con
 ### `element`
 **(Required)** A DOM element, jQuery element, or selector string to refer to the placeholder element.
```

Comment:
Should tweak the docs to stress this is a bit destructive, replacing the specified element.